test/cors_plug_test.exs

```diff
@@ -52,4 +51,16 @@ defmodule CORSPlugTest do
     assert Enum.member? conn.resp_headers,
                         {"access-control-allow-origin", "http://cors-plug.example"}
   end
+
+  test "exposed headers are returned" do
+    opts = CORSPlug.init(expose: ["content-range", "content-length", "accept-ranges"])
+    conn = conn(:options, "/")
+
+    conn = CORSPlug.call(conn, opts)
+    exposed_headers = conn.resp_headers
+      |> Enum.into(%{})
+      |> Map.get("access-control-expose-headers")
```

Comment:
Please use `Plug.conn.get_resp_header/2`. For muliline assigns please use the suggested form:

```Elixir
assign =
  val
  |> transform1
  |> transform2
  ...
```

Thanks!